bazel/repository_locations.bzl

```diff
@@ -25,10 +25,17 @@ REPOSITORY_LOCATIONS = dict(
         commit = "c0d77201039c7b119b18bc7fb991564c602dd75d",
         remote = "https://github.com/gcovr/gcovr",
     ),
+    com_github_opentracing_opentracing_cpp = dict(
+        commit = "713c15d40ae63185d2bec99bf3b03823967d7108",
+        remote = "https://github.com/opentracing/opentracing-cpp",
+    ),
     com_github_lightstep_lightstep_tracer_cpp = dict(
-        sha256 = "f7477e67eca65f904c0b90a6bfec46d58cccfc998a8e75bc3259b6e93157ff84",
-        strip_prefix = "lightstep-tracer-cpp-0.36",
-        urls = ["https://github.com/lightstep/lightstep-tracer-cpp/releases/download/v0_36/lightstep-tracer-cpp-0.36.tar.gz"],
+        commit = "c9d9215b5c3652c7eb5640903697d9a683e1df76",
+        remote = "https://github.com/lightstep/lightstep-tracer-cpp",
+        googleapis = dict(
```

Comment:
It's pretty confusing to have this extra level of nesting. How about giving googleapis its own location?

Reply:
Separated out. Is it ok to build target in the same function as lightstep?

Reply:
I think so. The functions only exist for human readability, and it makes sense to have all the lightstep stuff in one place.